### ISSUE TITLE
Training issue

### ISSUE THREAD

**Issue by Janebanana** [Hans1984/TAF#2]
Thank you for sharing the interesting work!

I tried to train the model on the datasets provided. Were the different scenes trained with the same setting? For example, I trained the model on real_all/scene_4 without changing anything and could not get reasonable results. Also, how to train the model on the real_dfd data? Should I remove the tone mapping network and how? Looking forward to hearing from you.

**Comment by Hans1984** [comment]
Hi, thanks for your attention. For 1, could you provide more information. For 2, yes,  in general, just remove the tmo part line 178 to 193, and change the loss function.

**Comment by Janebanana** [comment]
Thank you for your reply!

1. I retrained the model and the problem is resolved. I might have made some changes without attention.
2. Since the current blur image before tone mapping is HDR, do I also need to remove torch.exp(all_in_focus) to deal with that?
